description: upstream tries to use a unique_ptr that has already been moved
author: Michael Gilbert <mgilbert@debian.org>

--- a/services/resource_coordinator/coordination_unit/coordination_unit_base.cc
+++ b/services/resource_coordinator/coordination_unit/coordination_unit_base.cc
@@ -81,7 +81,8 @@ void CoordinationUnitBase::SetProperty(m
 // static
 CoordinationUnitBase* CoordinationUnitBase::PassOwnershipToGraph(
     std::unique_ptr<CoordinationUnitBase> new_cu) {
-  return new_cu->graph()->AddNewCoordinationUnit(std::move(new_cu));
+  CoordinationUnitGraph* graph = new_cu->graph();
+  return graph->AddNewCoordinationUnit(std::move(new_cu));
 }
 
 }  // namespace resource_coordinator
